docs/spelling_wordlist

```diff
@@ -446,6 +446,7 @@ permalink
 pessimization
 Peucker
 pgAdmin
+pgBouncer
```

Comment:
While you're here, it is styled "PgBouncer" rather than "pgBouncer". See http://www.pgbouncer.org/

Reply:
I'll pull this in. Thanks @pope1ni.